Packs/BeyondTrust_Password_Safe/ParsingRules/BeyondTrust_Password_Safe/BeyondTrust_Password_Safe.yml

```diff
@@ -0,0 +1,6 @@
+id: BeyondTrust_PasswordSafe_ParsingRule
```

Comment:
This ID is written differently than the one for modeling rules
Modeling: BeyondTrust_Password_Safe_ModelingRule
Parsing: BeyondTrust_PasswordSafe_ParsingRule (vs BeyondTrust_Password_Safe_ParsingRule